sklearn/tests/test_multioutput.py

```diff
@@ -588,3 +588,25 @@ def test_multioutput_estimator_with_fit_params(estimator, dataset):
     estimator.fit(X, y, some_param=some_param)
     for dummy_estimator in estimator.estimators_:
         assert 'some_param' in dummy_estimator._fit_params
+
+
+def test_regressor_chain_w_fit_params():
+    rng = np.random.RandomState(0)
+    X, y = datasets.make_regression(n_targets=3)
+    weight = rng.rand(y.shape[0])
+
+    class MySGD(SGDRegressor):
+
+        def fit(self, X, y, **fit_params):
+            self.sample_weight = fit_params['sample_weight']
+            super().fit(X, y, **fit_params)
+
+    model = RegressorChain(MySGD())
```

Comment:
```suggestion
    model = RegressorChain(MySGD(), MySGD())
```

Reply:
second parameter is `order` know. Hence I am getting `TypeError: 'MySGD' object is not iterable`, if I add this.

Reply:
Right my bad this isn't needed